release/initial-patches.sh

```diff
@@ -145,12 +145,24 @@ update_version() {
   git add "${BASEDIR}/.mvn/maven.config"
 }
 
+test_gpg_signing() {
+  declare binary
+  echo "**** testing gpg signing"
```

Comment:
Is there something special about the asterisks ?

Reply:
Just a "hey look at me!" sort of thing to highlight that if things are broken, this one is the first one to look at.